gamemode/modules/fadmin/fadmin/logging/sv_init.lua

```diff
@@ -11,12 +12,13 @@ FAdmin.StartHooks["Logging"] = function()
 
 		return true, OnOff
 	end)
+	logging = GetConVar("FAdmin_logging")
 end
 
 local LogFile
 function FAdmin.Log(text, preventServerLog)
 	if not text or text == "" then return end
-	if not GetConVar("FAdmin_logging"):GetBool() then return end
+	if not logging:GetBool() then return end
```

Comment:
Is it possible that any hook gets called before logging is set? Might want to be sure of that.